x-pack/plugins/secops/public/components/flyout/pane/index.tsx

```diff
@@ -21,25 +21,22 @@ interface FlyoutPaneProps {
   width: number;
 }
 
-/** SIDE EFFECT: This container has selectors that override EUI flyout styles  */
 const EuiFlyoutContainer = styled.div<{ headerHeight: number; width: number }>`
-  & > span > div {
-    .euiFlyout {
-      min-width: 150px !important;
-      width: ${({ width }) => `${width}px !important`};
-    }
-    .euiFlyoutHeader {
-      align-items: center !important;
-      display: flex !important;
-      flex-direction: row !important;
-      height: ${({ headerHeight }) => `${headerHeight}px !important`};
-      max-height: ${({ headerHeight }) => `${headerHeight}px !important`};
-      overflow: hidden;
-    }
-    .euiFlyoutBody {
-      overflow-y: hidden !important;
-      padding: 10px 24px 24px 24px !important;
-    }
+  .timeline-flyout {
+    min-width: 150px;
+    width: ${({ width }) => `${width}px`};
+  }
+  .timeline-flyout-header {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    height: ${({ headerHeight }) => `${headerHeight}px`};
+    max-height: ${({ headerHeight }) => `${headerHeight}px`};
+    overflow: hidden;
+  }
+  .timeline-flyout-body {
+    overflow-y: hidden;
+    padding: 10px 24px 24px 24px;
```

Comment:
Wow, this looks so much cleaner. The `!important` is gone and the multiple levels of CSS overrides are gone as well. This looks lots better. 👍